components/schemas/scoped-variables/RawSource.yml

```diff
@@ -13,12 +13,22 @@ properties:
   details:
     type: object
     required:
-      - string
+      - value
       - blob
     properties:
-      string:
+      value:
         type: string
         description: The value of the source.
       blob:
         type: boolean
         description: A boolean where true represents the text the user is entering will be multi line.
+      secret:
+        nullable: true
+        type: object
+        properties:
+          iv:
+            nullable: true
```

Comment:
Not nullable. It is omitempty in the struct so if it were to be 'nil' on the platform it will be omitted.
![image](https://github.com/cycleplatform/api-spec/assets/5110855/331fe0e2-e18f-4c37-99cf-92be2dddbd0c)